common/source/core/Registry.hpp

```diff
@@ -74,11 +94,14 @@ class Registry : public ISerializable {
 
 		const Recipe *getRecipe(const Inventory &inventory) const;
 
+		const Biome &getBiome(std::size_t id) const { return m_biomes.at(id); }
```

Comment:
Please use a `u32` here

Reply:
Note: Block and Item use a `std::size_t` but they will be replaced by a `u32` too in the future